packages/connect-conformance/package.json

```diff
@@ -3,21 +3,22 @@
   "version": "1.4.0",
   "private": true,
   "type": "module",
-  "main": "./dist/cjs/index.js",
+  "main": "./dist/cjs/src/index.js",
   "exports": {
     ".": {
-      "import": "./dist/esm/index.js",
-      "require": "./dist/cjs/index.js"
+      "import": "./dist/esm/src/index.js",
+      "require": "./dist/cjs/src/index.js"
     }
   },
   "bin": {
-    "connectconformance": "bin/connectconformance"
+    "connectconformance": "bin/connectconformance.cjs"
```

Comment:
I was consistently getting this error before:

> TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension "" for /Users/.../packages/connect-conformance/bin/connectconformance. Loading extensionless files is not supported inside of "type":"module" package.json contexts. The package.json file /Users/.../packages/connect-conformance/package.json caused this "type":"module" context. Try changing /Users/.../packages/connect-conformance/bin/connectconformance to have a file extension. Note the "bin" field of package.json can point to a file with an extension, for example {"type":"module","bin":{"connectconformance":"./bin/connectconformance.js"}}

Adding a .js or .mjs extension means we have to use an import assertion for package.json and compile with module = nodenext.

Seems much more simple to use CJS instead.